app/controllers/v0/sign_in_controller.rb

```diff
@@ -161,7 +164,9 @@ def revoke
     end
 
     def revoke_all_sessions
-      SignIn::RevokeSessionsForUser.new(user_account: @current_user.user_account).perform
+      session = SignIn::OAuthSession.find_by(handle: @access_token.session_handle)
+      user_account = UserAccount.find(session.user_account_id)
```

Comment:
you can just do `session.user_account` to get the `user_account` off the session